.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,62 @@
+# Semi-automated Nx releases where pre-created tags gets published to NPM.
+# The tags has been created locally using `nx release-cli`.
+
+name: Publish package
+
+on:
+  push:
+    tags:
+      - '*-*.*.*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      # Assume tag format '<package>-<version>'
+      - name: Extract package name
+        id: extract
+        run: |
+          TAG=${GITHUB_REF#refs/tags/}
+          PACKAGE=${TAG%-*}
+          echo "package=$PACKAGE" >> $GITHUB_OUTPUT
+
+        # Projects with target 'nx-release-publish' can be published
+      - name: Verify publishable package
+        run: |
+          PROJECTS=$(pnpm nx show projects --with-target nx-release-publish)
+          if echo "$PROJECTS" | grep -q "^${{ steps.extract.outputs.package }}$"; then
+            echo "Package '${{ steps.extract.outputs.package }}' is valid for publishing"
+          else
+            echo "'${{ steps.extract.outputs.package }}' is not a project configured for publishing"
+            echo "Available publishable projects:"
+            echo "$PROJECTS"
+            exit 0
```

Comment:
`exit 1`?

Reply:
Don't want it to fail just because it's not a publishable package, or some other tag with the same pattern